bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -200,109 +194,106 @@ public void run() {
     connectionStartupExecutor.shutdown();
   }
 
-  protected static ExecutorService createDefaultBatchPool() {
-    return Executors
-        .newCachedThreadPool(ThreadPoolUtil.createThreadFactory(BATCH_POOL_THREAD_NAME));
-  }
-
-  protected static EventLoopGroup createDefaultEventLoopGroup() {
-    return new NioEventLoopGroup(0, ThreadPoolUtil.createThreadFactory(GRPC_EVENTLOOP_GROUP_NAME));
-  }
-
-  protected static ScheduledExecutorService createDefaultRetryExecutor() {
-    return Executors.newScheduledThreadPool(RETRY_THREAD_COUNT,
-      ThreadPoolUtil.createThreadFactory(RETRY_THREADPOOL_NAME));
-  }
-
   private BigtableDataClient dataClient;
   private BigtableTableAdminClient tableAdminClient;
   private BigtableClusterAdminClient clusterAdminClient;
 
   private final BigtableOptions options;
-  private final ExecutorService batchPool;
-  private final boolean terminateBatchPool;
-  private final EventLoopGroup elg;
-  private final ScheduledExecutorService scheduledRetries;
   private final List<Closeable> clientCloseHandlers = Collections
       .synchronizedList(new ArrayList<Closeable>());
   private final ImmutableList<HeaderInterceptor> headerInterceptors;
 
   public BigtableSession(BigtableOptions options) throws IOException {
-    this (options, null, null, null);
-  }
-
-  public BigtableSession(BigtableOptions options, ExecutorService batchPool) throws IOException {
-    this(options, batchPool, null, null);
-  }
-
-  public BigtableSession(BigtableOptions options, @Nullable ExecutorService batchPool,
-      @Nullable EventLoopGroup elg, @Nullable ScheduledExecutorService scheduledRetries)
-      throws IOException {
     Preconditions.checkArgument(
         !Strings.isNullOrEmpty(options.getProjectId()), PROJECT_ID_EMPTY_OR_NULL);
-    Preconditions.checkArgument(
-        !Strings.isNullOrEmpty(options.getZoneId()), ZONE_ID_EMPTY_OR_NULL);
+    Preconditions.checkArgument(!Strings.isNullOrEmpty(options.getZoneId()), ZONE_ID_EMPTY_OR_NULL);
```

Comment:
nit: Not sure why this changed here but I slightly prefer it the old way, matching the other checkts. Up to you.